common/version/info_generated.go

```diff
@@ -0,0 +1,11 @@
+// Code generated by ziti-ci. DO NOT EDIT.
+
+package version
+
+const (
+	Version   = "v0.9.0"
+	Revision  = "ea556fc18740"
+	Branch    = "travis-build"
+	BuildUser = "Paul Lorenz"
```

Comment:
will ci overwrite this?

Reply:
yes. Right now it will only be reflected in the tag. Considering having master be updated as well, so even if you don't run it locally the version will be closer to accurate